.github/integration/scripts/charts/dependencies.sh

```diff
@@ -22,6 +22,16 @@ if [ "$1" == "local" ]; then
                 echo "yq not installed, get it from here: https://github.com/mikefarah/yq/releases/latest"
                 exit 1
         fi
+
+        if [ ! "$(command jq)" ]; then
```

Comment:
`command jq`  tries to run the commands instead of checking if they exist. This will fail if the command requires arguments, so as for this case.

It's better to change all usage of `command aaa` to improve the robustness.